ee/libcglue/src/cwd.c

```diff
@@ -149,9 +149,15 @@ int __path_absolute(const char *in, char *out, int len)
 			return -1;
 	} else if(dr > 0 && in[dr - 1] == ':') {
 		/* It starts with "drive:", so it's already absoulte, however it misses the "/" after unit */
-		strncpy(out, in, dr);
-		out[dr] = '/';
-		strncpy(out + dr + 1, in + dr, len - dr - 1);
+		/* Just do it if drive: is different than rom0:, rom0 doesn't expect to have an slask */
+		if(strncmp(in, "rom0:", 5) == 0) {
```

Comment:
Should also handle the case of `rom1`, `rom2`, ...

Reply:
I have updated PR from `rom0` to `rom9`